.github/workflows/build.yml

```diff
@@ -307,9 +307,7 @@ jobs:
       # No need to do the PPA build for both Ubuntu versions
       if: matrix.os == 'ubuntu-20.04'
       run: |
-        if [[ "${{ github.ref }}" == "refs/heads/2.3" ]] && [[ "${{ github.repository }}" == "mixxxdj/mixxx" ]]; then
-          CPACK_ARGS="-D DEB_UPLOAD_PPA=ppa:mixxx/mixxxbetas"
```

Comment:
shouldn't we just change "refs/heads/2.3" to "refs/heads/2.4"? Why are we removing the beta upload instead?

Reply:
This is a PR against the 2.3 branch, so the 2.4 condition will never be true. 

We also don't want the 2.4 branch to populate the mixxxbeta stream before we have a beta. 

My idea was to add it manualy. 

We may think about to check git describe if it is a beta and select the destination PPA accordingly.

Reply:
Ah right, that makes sense. Can you take care of merging to main then?
`

Reply:
I have considered the git describe idea. It will unfortunately not work alone, because feature branches will also send to the PPA, because they will not have an extra tag. A working solution is hard to maintain and test. So I would like to stick with the branch based manual assignment like we have here for now. 

I will take care merging this to the main branch as well.

Reply:
So for the 2.4 beta, we just revert this PR on `main` and fix mentions of `2.3` to `2.4`?

Reply:
Not exactly, we need to wait until we have tagged a 2.4-beta in the 2.4 branch and than re-add this code for the 2.4 branch there.

Reply:
Right, of course.